Week-5/Work & HW/index/blackjack.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Black Jack</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <style>
+    #alert {
+      font-size: 300px;
+    }
+    </style>
+  </head>
+  <body>
+    <div id="top">
+    </div>
+    <h1>Your Hand</h1>
+    <div id="div">
+    </div>
+    <button id="hit">Hit</button>
+    <button id="stay">Stand</button>
+    <div id="div2">
+      <h1>Dealer</h1>
+    </div>
+
+    <script>
+      let cards = [];
+      let cardsimages = [];
+      let cards2 = {};
+      cards2.cards = [];
+      cards2.total;
+      let cardsTotal;
+      let cards2images = [];
+      let checkNum = (num, array) => {
+        if (num.includes("2") || num.includes("3") || num.includes("4") || num.includes("5") || num.includes("6") || num.includes("7") || num.includes("8") || num.includes("9") || num.includes("10")) {
```

Comment:
why num.includes? num is really CARD in this context right? why not just `===` or something more concise?

I think you can also create an array of the values you want: `["2", "3", "4", "5", ...].includes(card)` which might be even more concise